clinica/cmdline.py

```diff
@@ -7,10 +7,10 @@
 """
 import click
 
-from .engine.template import cli as generate_cli
-from .iotools.converters.cli import cli as convert_cli
-from .iotools.utils.cli import cli as iotools_cli
-from .pipelines.cli import cli as run_cli
+from clinica.engine.template import cli as generate_cli
+from clinica.iotools.converters.cli import cli as convert_cli
+from clinica.iotools.utils.cli import cli as iotools_cli
+from clinica.pipelines.cli import cli as run_cli
```

Comment:
That should not be necessary. You should fix your `pylint` setup instead.

Reply:
the issue is not with the pylint but with the python debugger which can not find this relative path. It seems to be an issue only with this file (as it is top level). 
Is there a strong reason against being explicit with the path here?

Reply:
If it's only this file then ok (though debugging works on PyCharm so there might be something we are missing).